lib/services/jira/jira_mapped_fields.rb

```diff
@@ -67,6 +74,19 @@ def aha_type_to_string(aha_type, aha_value)
       aha_value.to_s
     end
   end
+
+  def aha_type_to_user(aha_type, field)
+
+    if aha_type == "string"
+      user_resource.picker(field.value.strip).try(:[], :key)
+    else
+      field.email_value.each do |email|
+        potential_key = user_resource.picker(email).try(:[], :key)
+        return potential_key  unless potential_key.nil?
+      end
+      nil
+    end
+  end
```

Comment:
@taltas I wonder if this could be simplified a bit with a little type coercion. Whenever in ruby I see code that's handling a scalar value case and an `Array` case where the elements of the Array are handled the same as the scalar case, coercing to an `Array` can help by making it so there is only a single case. Something like this:

```ruby
def aha_type_to_user(aha_type, field)
  Array(field.email_value).find do |email|
    user_resource.picker(email.strip).try(:[], :key)
  end
end
```

By coercing `field.email_value` to an `Array` a few things would happen.
  1. If it's a `String`, it'll wrap it in an `Array`: `Array("foo") #=> ["foo"]`
  2. If it's already an `Array`, it's just an identity function so it's a noop: `Array(["foo"]) #=> ["foo"]`
  3. And lastly, we get a nice guard against `nil` aha_type here. Not that it should happen, but being defensive via coercion can be nice. In the `nil` case, it'd do this: `Array(nil) #=> []`

Also, by using `Enumerable#find` as opposed to `#each`, we're able to clean up the inside of the block a bit.

The one thing that is different from the original method is that we're now always calling `String#strip` on the `field.value`. Not sure if that's desirable.